qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -729,20 +750,8 @@ def multiple_trigger_acquisition(self, mV_range, memsize, seg_size, posttrigger_
         self.general_command(pyspcm.M2CMD_CARD_START |
                              pyspcm.M2CMD_CARD_ENABLETRIGGER | pyspcm.M2CMD_CARD_WAITREADY)
 
-        # setup software buffer
-        buffer_size = ct.c_int16 * memsize * numch
-        data_buffer = (buffer_size)()
-        data_pointer = ct.cast(data_buffer, ct.c_void_p)
-
-        # data acquisition
-        self._def_transfer64bit(
-            pyspcm.SPCM_BUF_DATA, pyspcm.SPCM_DIR_CARDTOPC, 0, data_pointer, 0, 2 * memsize * numch)
-        self.general_command(pyspcm.M2CMD_DATA_STARTDMA |
-                             pyspcm.M2CMD_DATA_WAITDMA)
-
-        # convert buffer to numpy array
-        data = ct.cast(data_pointer, ct.POINTER(buffer_size))
-        output = np.frombuffer(data.contents, dtype=ct.c_int16)
+        # convert transfer data to numpy array
+        output = self._transfer_buffer_numpy(memsize, numch, bytes_per_sample=2)

```

Comment:
should bytes_per_sample also be an argument of the multiple_trigger_acquisition method? because now there is no way to run multiple_trigger_acquisition for bytes_per_sample==4.

Reply:
The `multiple_trigger_acquisition` does not generated values with 4 bytes precision.